samples/validate/validate.cpp

```diff
@@ -405,3 +409,45 @@ bool MyDialog::TransferDataToWindow()
     return r;
 }
 
+void MyDialog::OnValidationFailed(wxValidationEvent& event)
+{
+    if ( event.GetEventType() == wxEVT_TXT_VALIDATE )
```

Comment:
I don't really like these tests for the event types. Why do we need them? Just handle errors from both validators in the same way. And the "Empty strings not allowed" string should be part of the message, not come from this code.

Reply:
This is just to show how we can replace the event error message (here "Required information entry is empty.") with our own, which might be useful for some applications.